ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1670,6 +1686,138 @@ - (ORKQuestionType) questionType {
 @end
 
 
+#pragma mark - ORKTextScaleAnswerFormat
+
+@implementation ORKTextScaleAnswerFormat {
+    NSNumberFormatter *_numberFormatter;
+}
+
+- (Class)questionResultClass {
+    return [ORKScaleQuestionResult class];
+}
+
+- (instancetype)init {
+    ORKThrowMethodUnavailableException();
+    return nil;
+}
+
+- (instancetype)initWithTextChoices:(NSArray<ORKTextChoice *> *)textChoices
+                       defaultIndex:(NSInteger)defaultIndex
+                           vertical:(BOOL)vertical {
+    self = [super init];
+    if (self) {
+        _textChoices = textChoices;
```

Comment:
Don't forget to also use `copy` here after changing the property attribute, as mentioned by Yuan below.